hilti/src/rt/library.cc

```diff
@@ -1,14 +1,51 @@
 // Copyright (c) 2020 by the Zeek Project. See LICENSE for details.
 
 #include <dlfcn.h>
+#include <nlohmann/json.hpp>
```

Comment:
Let's instead push `hilti/json.h` into `hilti/rt` and include it here so we can still suppress diagnostics in `nlohmann/json.hpp`. We'd also move the dep on `nlohmann_json` from `hilti` to `hilti-rt` again for that.